pyo3-macros-backend/Cargo.toml

```diff
@@ -21,3 +21,6 @@ proc-macro2 = { version = "1", default-features = false }
 version = "1"
 default-features = false
 features = ["derive", "parsing", "printing", "clone-impls", "full", "extra-traits"]
+
+[build-dependencies]
+pyo3-build-config = { path = "../pyo3-build-config", version = "0.14.0-alpha.0" }
```

Comment:
The main Cargo.toml has an `=` constraint here.

Also, is there a script to update these on version bumps?

Reply:
Thanks, fixed.

Sadly there's not, up until this point I've been bumping everything manually as part of a release PR. It's not too bad to do it with a find-and-replace, though a script would be a welcome improvement 😂